package.json

```diff
@@ -15,5 +15,8 @@
   "bugs": {
     "url": "https://github.com/amillena/Flashcard-Generator/issues"
   },
-  "homepage": "https://github.com/amillena/Flashcard-Generator#readme"
+  "homepage": "https://github.com/amillena/Flashcard-Generator#readme",
+  "dependencies": {
```

Comment:
You wanna make sure your dependencies get saved to your `package.json` so that other developers can clone your repo and simply run `npm install` to get everything they need to get up and running.